pages/linux/xcursorgen.md

```diff
@@ -0,0 +1,21 @@
+# xcursorgen
+
+> Create an X cursor file from a collection of PNG images.
+> If `--prefix` is omitted, the image files must be located in the working directory.
+> More information: <https://manned.org/xcursorgen.1>.
+
+- Create an X cursor file using a config file:
+
+`xcursorgen {{path/to/config.cursor}} {{path/to/output_file}}`
+
+- Create an X cursor file using a config file and specify the path to the image files:
+
+`xcursorgen --prefix {{path/to/image_directory/}} {{path/to/config.cursor}} {{path/to/output_file}}`
+
+- Create an X cursor file using a config file and write the output to standard output:
+
+`xcursorgen {{path/to/config.cursor}}`
+
+- Display version:
```

Comment:
Is this example really needed? Unless it's important for the user to know what version of the program they're using (e.g. to check for support for a feature they need), I'm unsure it's really adding something to the page. 8 examples per page is a _limit_, not a _target_ :sweat_smile:

Reply:
Yeah, you're right, it's kind of distracting. I'll remove it.